cli/connect/console.go

```diff
@@ -233,7 +235,7 @@ func getExecutor(console *Console) func(string) {
 		}
 
 		var data string
-		if _, err := console.conn.Eval(consoleEvalFuncBody, args, opts); err != nil {
+		if _, err := console.conn.Eval(evalFuncBody, args, opts); err != nil {
```

Comment:
It seems like with the `evalFuncBody` `\set language sql` and `\set delimiter` won't work anymore. So we need tests for the cases + update `consoleEvalFuncBody` in the `console.eval` implementation manner too.

Reply:
Updated.